script.js

```diff
@@ -1 +1,42 @@
-console.log('Yes maam');
+//to do manipulate the DOM
+
+const computerPlay = () => {
+    const computerInput = Math.floor(Math.random() * Math.floor(3));
+    if (computerInput === 0) {
+        return 'rock';
+    }
+    if (computerInput === 1) {
+        return 'paper';
+    }
+    if (computerInput === 2) {
+        return 'scissors';
+    }
+}
+
+const humanSelection = prompt('Rock paper or scissors?');
```

Comment:
Just a suggestion, you can call toLowerCase() on a string literal. Try it like this: 
`const humanSelection = prompt('Rock paper or scissors?'.toLowerCase());`
That way, you'll save a line of code, and D.R.Y